pshops_lib/check_account_no.js

```diff
@@ -0,0 +1,28 @@
+/*
+    Function: check_account_no.js
+    Purpose: checks if the input account number appears as a PShop account in Roster
+    Status: complete
+*/
+
+module.exports = function(accnum){
+    // Authentication against Roster
+    var rosterAPI = require('ext/Roster_v1_0_2/api');
```

Comment:
let's talk through maybe abstracting this out so it's not hardcoded (something to fix in my code as well!)